tracks/csharp/exercises/high-scores/mentoring.md

```diff
@@ -0,0 +1,35 @@
+### Reasonable solution
+
+#### using LINQ 
+```csharp
+using System.Collections.Generic;
+using System.Linq;
+
+public class HighScores
+{
+    // Fields
+    private readonly List<int> _scores; 
+
+    // Constructor
+    public HighScores(List<int> list)
+    {
+        _scores = list;
+    }
+
+    // Methods
+    public List<int> Scores() => _scores.ToList(); 
+    public int Latest() => _scores.Last();
+    public int PersonalBest() => _scores.Max();
+    public List<int> PersonalTopThree() => _scores.OrderByDescending(s => s).Take(3).ToList();
+}
+```
+
+### Common suggestions
+- The exercise focuses mainly on using two concepts: list as generic collection, combined with basic LINQ methods. 
```

Comment:
This doesn't seem like a suggestion. I recommend moving it to line 5! It's a good comment for the reasonable solution.

Reply:
done